nx/lib/nx.ex

```diff
@@ -5013,6 +5057,9 @@ defmodule Nx do
   end
 
   ## Unary ops
+  @allow_complex_type_unary_ops Nx.Shared.unary_math_funs()
+                                |> Keyword.keys()
+                                |> Kernel.--([:erf, :erfc, :erf_inv])
```

Comment:
Wait, is `erf` not a complex operation? So why do we have Complex.erf? For completeness?

Reply:
There is a complex definition for erf, as per: https://mathworld.wolfram.com/Erf.html
I left it unimplemented for now and chose to not support it in Nx until then. Perhaps we should remove it altogether (and bring the erf and friends implementations back to Nx.Shared)

Reply:
I guess the question is: is it supported by torch/xla? If so, I think we should have it working here and raise inside the `Complex` library.